msal4j-sdk/src/samples/msal-web-sample/src/main/java/com/microsoft/azure/msalwebsample/CookieHelper.java

```diff
@@ -32,6 +32,7 @@ static void removeStateNonceCookies(HttpServletResponse httpResponse){
         Cookie stateCookie = new Cookie(MSAL_WEB_APP_STATE_COOKIE, "");
         stateCookie.setMaxAge(0);
 
+        // CodeQL [java/insecure-cookie]: Suppressing CodeQL warning since this is just a sample
```

Comment:
@Avery-Dunn - can we revisit this? Our samples should not use insecure practices.

Reply:
This is just a simpler version of the "real" sample meant to help with manual E2E testing rather than teach something, however it turns out the same code is in the real sample: https://github.com/Azure-Samples/ms-identity-msal-java-samples/blob/main/1-server-side/msal-b2c-web-sample/src/main/java/com/microsoft/azure/msalwebsample/CookieHelper.java#L30

I've just created an issue about auditing the samples to find these sorts of insecure practices: https://github.com/AzureAD/microsoft-authentication-library-for-java/issues/911